hera_cal/smooth_cal.py

```diff
@@ -664,8 +662,8 @@ def freq_filter(self, filter_scale=10.0, tol=1e-09, window='tukey', skip_wgt=0.1
                                                      filter_scale=filter_scale, tol=tol, window=window,
                                                      skip_wgt=skip_wgt, maxiter=maxiter, **win_kwargs)
             # flag all channels for any time that triggers the skip_wgt
-            for i, info_dict in enumerate(info):
-                if info_dict.get('skipped', False):
+            for i in info['status']['axis_1']:
```

Comment:
The keys of this dictionary are integer indices? You're positive it shouldn't be for i in range(len(...?

Reply:
The keys are integer indices so I'm happy to iterate through them. I am open to suggestions if you think this system is opaque.